railties/lib/rails/generators/rails/app/templates/Gemfile.tt

```diff
@@ -59,8 +59,7 @@ end
 <%- if depends_on_system_test? -%>
 group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
-  gem "capybara"
-  gem "selenium-webdriver"
-  gem "webdrivers"
+  gem "capybara", ">= 3.39"
+  gem "selenium-webdriver", ">= 4.11.0"
```

Comment:
```suggestion
  gem "capybara"
  gem "selenium-webdriver"
```

New applications don't have a Gemfile.lock yet, so they will always get the most up-to-date gem.

Reply:
If `webdrivers` is necessary in the version that support Ruby 2.7 we will need to conditionally add that gem when the app is generated on that Ruby version.